.buildkite/pipeline.yml

```diff
@@ -0,0 +1,23 @@
+steps:
+  - label: "analyzegc"
+    commands:
+      - apt-get update
+      - apt-get install -y build-essential libatomic1 python python3 gfortran perl wget m4 cmake pkg-config curl
+      - make -j 6 -C deps install-llvm install-clang install-llvm-tools install-libuv install-utf8proc install-unwind
+      - make -j 6 -C test/clangsa
+      - make -j 6 -C src analyzegc
+    agents:
+      queue: "juliacpu" # this should be julia -- also in pipeline settings
```

Comment:
The `juliacpu` queue is the one using the Dockerized JuliaGPU set-up from https://github.com/JuliaGPU/buildkite. Maybe the `julia` queue should be similarly dockerized (so that we can safely run it on PRs) but with a different image, including the packages installed here. I'm not sure who set-up or is using that queue, @staticfloat or @christopher-dG?

Reply:
That was me, from my older attempts.

Reply:
Is it okay to make those changes later, and merge this PR with the `juliacpu` queue? Or should those changed be made before merging this PR?

Reply:
The `juliacpu` queue is not to busy. I would like @staticfloat approval before we move forward.